src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costsPerDay = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  return (days >= 3 && days <= 6)
+    ? ((days * costsPerDay) - smallDiscount) : (days >= 7)
+      ? ((days * costsPerDay) - bigDiscount) : (days * costsPerDay);
```

Comment:
Ternary operator should not be used this way - it's making your code overcomplicated.

I would suggest using structure like:

```
if (condition) {
  return x
};

return y;
```